boards/t400-maximized-debug/t400-maximized-debug.config

```diff
@@ -0,0 +1,61 @@
+# Configuration for a t400 running non-Qubes OSes with USBDEBUG activated under coreboot config
+#
+# Deactivated to fit in coreboot's CONFIG_CBFS_SIZE=0x7FCFFF:
+# dropbear support(ssh client/server)
```

Comment:
Could be reactivated. When looking at output of builds, there is still  of space for 8mb ROMs.
Note that really soon, bash will be included in roms, taking 150kb of CBFS space.